src/client/components/CheckReviews.js

```diff
@@ -0,0 +1,51 @@
+import React,{useState,useEffect}from 'react'
+
+function CheckReviews({idOfMeal}) {
+    const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const [reviews,SetReviews]=useState([]);
```

Comment:
Remember to format your code. This is cheap. I use ctrl-alt-F on windows in vscode